### DISCUSSION TITLE
TextBox Text property updates only when user selects text suggestion or presses space on android

### DISCUSSION THREAD

**Discussion by JenyaRostov** [AvaloniaUI/Avalonia#10952] [Q&A]
Title
How do i prevent that? How do i make Text property update every time user types a character, not when they select text suggestion? Or maybe i can disable it altogether?
![ezgif-3-bf16d2a922](https://user-images.githubusercontent.com/52300935/230722675-99e58460-b2f2-48ec-b0eb-d91430832b32.jpg)

**Comment by maxkatz6** [comment]
Which version of avalonia do you use? This issue was solved quite a while ago last year in previews.

**Reply by JenyaRostov** [reply]
Im using preview 4 from avalonia.xplat template

**Reply by timunie** [reply]
@JenyaRostov please update to preview 6 or newer. Best is nightly build

**Reply by JenyaRostov** [reply]
is there xplat template with it? i tried updating but some stuff got changed and i was getting errors so i reverted to preview4

**Reply by timunie** [reply]
Read breaking changes wiki. Also you may have to update the templates , i.e. reinstall them.

**Reply by JenyaRostov** [reply]
thank you, will try that
